Osiris/Hacks/Aimbot.cpp

```diff
@@ -94,38 +94,46 @@ static bool canScan(Entity* localPlayer, Entity* entity, const Vector& destinati
 
 void Aimbot::run(UserCmd* cmd) noexcept
 {
-    const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
-    if (localPlayer->nextAttack() > memory.globalVars->serverTime())
-        return;
+	const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+	if (localPlayer->nextAttack() > memory.globalVars->serverTime())
+		return;
 
-    const auto activeWeapon = localPlayer->getActiveWeapon();
-    if (!activeWeapon || !activeWeapon->clip())
-        return;
+	const auto activeWeapon = localPlayer->getActiveWeapon();
+	if (!activeWeapon || !activeWeapon->clip())
+		return;
 
-    auto weaponIndex = getWeaponIndex(activeWeapon->itemDefinitionIndex2());
-    if (!weaponIndex)
-        return;
+	auto weaponIndex = getWeaponIndex(activeWeapon->itemDefinitionIndex2());
+	if (!weaponIndex)
+		return;
 
-    auto weaponClass = getWeaponClass(activeWeapon->itemDefinitionIndex2());
-    if (!config.aimbot[weaponIndex].enabled)
-        weaponIndex = weaponClass;
+	auto weaponClass = getWeaponClass(activeWeapon->itemDefinitionIndex2());
+	if (!config.aimbot[weaponIndex].enabled)
+		weaponIndex = weaponClass;
 
-    if (!config.aimbot[weaponIndex].enabled)
-        weaponIndex = 0;
+	if (!config.aimbot[weaponIndex].enabled)
+		weaponIndex = 0;
 
-    if (!config.aimbot[weaponIndex].betweenShots && activeWeapon->nextPrimaryAttack() > memory.globalVars->serverTime())
-        return;
+	if (!config.aimbot[weaponIndex].betweenShots && activeWeapon->nextPrimaryAttack() > memory.globalVars->serverTime())
+		return;
```

Comment:
no changes here, i think it was automaticly caused by visual studio while compiling somehow...